### ISSUE TITLE
Availability of 2D Experiments?

### ISSUE THREAD

**Issue by GloryyrolG** [swyoon/normalized-autoencoders#9]
Hi Sangwoong @swyoon ,

May I ask is there a share of the good 2D illustration (CD, PCD, etc.) in the paper?

Thanks,

**Comment by swyoon** [comment]
Dear @GloryyrolG ,

I have updated materials for 2d density estimation experiment under the directory `2d_density_estimation`.

If you want to reproduce training, please see `nae_training_methods.ipynb`. 

For visualization, please see `draw_2d_density_estimation_figure.ipynb` and `draw_nae_training_methods.ipynb`.

I have made a slight modification when I bring my classes to this repository, so things may not be *exactly* the same.

However, the training code seems to work.

Please let me know if there's anything strange.

Always thank you for your attention.
